src/lib.rs

```diff
@@ -1330,7 +1326,13 @@ mod tests {
     test_ivc_nontrivial_with_compression_with::<
       bn256::Point,
       grumpkin::Point,
-      S<bn256::Point, EE<_>>, // SZM<bn256::Point, halo2curves::bn256::Bn256>,
+      S<bn256::Point, ZM<halo2curves::bn256::Bn256>>,
+      S<grumpkin::Point, EE<_>>,
+    >();
+    test_ivc_nontrivial_with_compression_with::<
```

Comment:
Why do we need this test instance to run twice using exactly same parameters? Shouldn't it better to use:
```
test_ivc_nontrivial_with_compression_with::<
      bn256::Point,
      grumpkin::Point,
      SPrime<bn256::Point, ZM<halo2curves::bn256::Bn256>>,
      SPrime<grumpkin::Point, EE<_>>,
    >();
```
instead, in order to use pp-spartan's proving / verifications for one of test instances?

Reply:
Yep, that's silly!